common/lib/xmodule/xmodule/course_module.py

```diff
@@ -682,7 +682,7 @@ class CourseFields(object):
         ),
         scope=Scope.settings,
         # Ensure that courses imported from XML keep their image
-        default="images_course_image.jpg"
+        default=""
```

Comment:
"XML integration" meaning that courses authored in XML have their course image I think at a specific place in their course structure. I don't quite recall how the XML courses work, but I believe we still have XML courses on production and we should evaluate whether or not this change would make it so that XML courses no longer display their course image. If they don't, would this be a problem?